requirements.txt

```diff
@@ -0,0 +1,8 @@
+dandi
```

Comment:
Do you use this import anywhere?

Reply:
Thanks.  Its for the `extract_metadata.py` script.  Out of scope for this pull request but still needed so will keep in.